### PR TITLE
VSHE Release note 2024-05-13.md

### PR DESCRIPTION
**Pull request by camronmartinez**

### CODE REVIEW

File: content/verification/veteran-verification/release-notes/2024-05-13.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+We updated the results returned for disability ratings. They will now exclude ratings that do not have a “begin” date.
+Added fields  (underline)
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-05-13.md:2]: What does "Added fields (underline)" mean? I need more clarity here.

File: content/verification/veteran-verification/release-notes/2024-05-13.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+We added a second (optional) diagnostic type code field in the response section for individual ratings attributes. Field name: hyph_diagnostic_type_code.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-05-13.md:3]:
```suggestion
We added a second (optional) diagnostic type code field in the response section for individual ratings attributes. The field name is hyph_diagnostic_type_code.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]

File: content/verification/veteran-verification/release-notes/2024-05-13.md (hunk 1 above)
Comment (camronmartinez) [reply content/verification/veteran-verification/release-notes/2024-05-13.md:2]: Got some clarification from Lorri. She was just referencing that we added the new field **hyph_diagnostic_type_code** and she meant underscore. We can remove the line:
Added fields (underline)

File: content/verification/veteran-verification/release-notes/2024-05-13.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+We updated the results returned for disability ratings. They will now exclude ratings that do not have a “begin” date.
+
+We added a second (optional) diagnostic type code field in the response section for individual ratings attributes. The field name is hyph_diagnostic_type_code.
+If we receive the optional code from the disability rating source system, it will be included in the API response (directly after diagnostic_type_code).
+Additionally, a field labeled static indicator is now included in the API response for disability ratings. The values in this field are (Y)es or (N)o.
```
Comment (joeabraun) [inline content/verification/veteran-verification/release-notes/2024-05-13.md:5]:
```suggestion
Additionally, a field labeled static indicator is now included in the API response for disability ratings. The values in this field are Y for yes or N for no.
```

**Review by joeabraun — CHANGES_REQUESTED** [review]

**Review by joeabraun — APPROVED** [review]
Approved!

**Pull request merged**